### PR TITLE
add terraform kubectl provider config

### LINKED ISSUE #86: [Bug]: terraform bootstrap fails with kubectl error
### Welcome to AWS Blueprints for Crossplane!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/aws-samples/crossplane-aws-blueprints/issues) and didn't find any.

### AWS Blueprints for Crossplane Release version

latest

### What is your environment, configuration and the example used?

the out of the box example for terraform

### What did you do and What did you see instead?

normal deployment with terraform
```
git clone https://github.com/awslabs/crossplane-on-eks
cd bootstrap/terraform
terraform init
terraform apply -auto-approve
```
Errored out with
```
│ Error: aws-controller-config failed to create kubernetes rest client for update of resource: Get "https://xxx.gr7.us-east-1.eks.amazonaws.com/api?timeout=32s": dial tcp: lookup xxx.gr7.us-east-1.eks.amazonaws.com on [xxx:a601:a604:500::1]:53: no such host
│ 
│   with module.eks_blueprints_kubernetes_addons.module.crossplane[0].kubectl_manifest.aws_controller_config[0],
│   on .terraform/modules/eks_blueprints_kubernetes_addons/modules/kubernetes-addons/crossplane/main.tf line 20, in resource "kubectl_manifest" "aws_controller_config":
│   20: resource "kubectl_manifest" "aws_controller_config" {
```

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by csantanapr**
### What does this PR do?

Adds the kubectl terraform provider version and config

### Motivation

Fixes #86 

### More

- [x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)

- [ ] Yes, I have added a new example under [examples](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/examples) to support my PR

- [ ] Yes, I have updated the [docs](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/doc) for this feature

- [x] Yes, I have linked to an issue or feature request (applicable to PRs that solves a bug or a feature request)

**Note**:
 - Not all the PRs require examples and docs
 - We prefer small, well tested pull requests. Please ensure your pull requests are self-contained, and commits are squashed

### For Moderators
- [x] E2E Test successfully complete before merge?

### CODE REVIEW

**Comment by csantanapr** [comment]
@nabuskey or @nimakaviani please review when you have chance

**Review by candonov — APPROVED** [review]
lgtm

File: bootstrap/terraform/README.md [hunk 1]
```diff
@@ -90,6 +90,10 @@ kubectl get nodes
 ```shell script
 kubectl get pods -n crossplane-system
 ```
+#### Step8: Verify the names of the providerConfig resources for all providers installed
+```shell script
+kubectl get provider
```
Comment (nabuskey) [inline bootstrap/terraform/README.md:95]: This gets Crossplane providers, not ProviderConfigs.

**Review by nabuskey — APPROVED** [review]

File: bootstrap/terraform/README.md (hunk 1 above)
Comment (csantanapr) [reply bootstrap/terraform/README.md:106]: @nabuskey there is a difference between `kubectl get provider` and `kubectl ger providers`

The command `kubectl get provider` will returns all CRs that the corresponding CRD `category` has `provider` like you can see in the kubernetes providerconfig CRD https://github.com/crossplane-contrib/provider-kubernetes/blob/main/package/crds/kubernetes.crossplane.io_providerconfigs.yaml#L14

I added both commands in the instructions

**Pull request merged**